src/main/java/com/artipie/repo/ConfigFile.java

```diff
@@ -76,19 +83,19 @@ public ConfigFile(final Key filename) {
      * @param storage Storage where the file with different extensions is checked for existence
      * @return True if a file with either of the two extensions exists, false otherwise.
      */
-    public CompletionStage<Boolean> existsIn(final Storage storage) {
-        final CompletionStage<Boolean> res;
-        if (this.isYamlOrYml() || this.matcher("extension").isEmpty()) {
+    public CompletableFuture<Boolean> existsIn(final Storage storage) {
```

Comment:
@olegmoz I replace `CompletionStage` with `CompletableFuture` because `StorageAliases#find` returns `CompletableFuture` but now realized that I can just use method `toCompletableFuture()` to convert.